elf/elf.py

```diff
@@ -153,6 +153,16 @@ def add_symbols_version(self):
         # symbols_version_requirement
         pass
 
+	def add_functions(self):
+		if hasattr(self.elf, "imported_functions"):
```

Comment:
Would it be possible to verify that self.elf.imported_functions and self.elf.exported_functions are not empty before creating the sections? If not, it create empty sections with empty json.
![image](https://user-images.githubusercontent.com/75089569/207949632-b32d7869-7bbb-4e2b-b3b1-1da0009e7b4e.png)

Reply:
Of course no problem and it's done